common/main/si.xml

```diff
@@ -9654,7 +9654,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<namePattern>{surname} {given-informal}</namePattern>
```

Comment:
this is for a few lines up.
In Sinhala, we should add _prefix_ in long, referring, formal
```
		<personName order="surnameFirst" length="long" usage="referring" formality="formal">
			<namePattern>{prefix} {surname} {given} {given2} {suffix}</namePattern>
		</personName>
```
see: https://cmb.ac.lk/honorary-degrees/ - si_Latn - formal presentation
https://www.parliament.lk/uploads/documents/paperspresented/annual-report-university-of-jaffna-2017-si.pdf - list of faculty